README.md

```diff
@@ -64,7 +69,12 @@ protected List<ReactPackage> getPackages() {
 ```
 </details>
 
-## Usage
+#### Using React Native >= 0.60
```

Comment:
You can remove this section. Jetifier for backwards compatibility will be needed when we update the library to Android X. Right now for `>= 0.60`, Jetifier will run automatically when users run `react-native run-android`.